### DISCUSSION TITLE
Piccolo Store

### DISCUSSION THREAD

**Discussion by dantownsend** [piccolo-orm/piccolo#477] [General]
Over the last few weeks I've been working on the Piccolo Store:

https://piccolo-store.com/

We've accumulated a tonne of Python knowledge while creating Piccolo and associated projects. Sometimes there's stuff I'd like to share which is too big for the [blog](https://piccolo-orm.com/blog/). So I thought about selling these handbooks on certain programming topics.

The first one is about documentation, as that's what I've been doing a really deep dive into over the last couple of months. I have ideas for some others too - databases / Piccolo, front end development etc.

The content will be reasonably priced, and any funds generated will help support development of the Piccolo ecosystem.

Thanks!

**Comment by sinisaos** [comment]
@dantownsend This is great. I hope it will be successful and that any funds will come in handy to support Piccolo.

**Reply by dantownsend** [reply]
Thanks, fingers crossed 🤞
